engine/worker/run.go

```diff
@@ -155,7 +155,7 @@ func (w *currentWorker) runJob(ctx context.Context, a *sdk.Action, buildID int64
 		return sdk.Result{
 			Status:  sdk.StatusFail.String(),
 			BuildID: buildID,
-			Reason:  "Unable to interpolate action parameters",
+			Reason:  err.Error(),
```

Comment:
i suggest: `"Unable to interpolate action parameters (%v)", err`

Reply:
no, because in the error of interpolate there is already `Unable to interpolate action parameters` so in fact the final sentence is for example : `Unable to interpolate action parameters: Invalid template format: template: input:17`